utils/parseHinfahrtRecon.ts

```diff
@@ -0,0 +1,87 @@
+import zlib from "node:zlib";
+import { z } from "zod/v4";
+
+export const parseHinfahrtReconCrude = (hinfahrtRecon: string) => {
+	const dirtyRegex = /A=1@O=.*?@a=\d+@/g;
+	const matches = hinfahrtRecon.match(dirtyRegex);
+
+	if (matches === null) {
+		throw new Error("Can't process vbid: hinfahrtRecon regex failed");
+	}
+
+	return [matches[0], matches.at(-1)!] as const;
+};
+
+export const parseHinfahrtRecon = (hinfahrtRecon: string) => {
+	// hinfahrtRecon ist ein verfluchtes format.
```

Comment:
Comments like these are fun and all but they don't belong to a codebase imo :D Also it's in German (making contributing less accessible for internationals) and doesn't ellaborate on why the format is supposedly terrible